CRM/Upgrade/Incremental/php/FiveSixtySix.php

```diff
@@ -29,6 +29,41 @@ class CRM_Upgrade_Incremental_php_FiveSixtySix extends CRM_Upgrade_Incremental_B
    */
   public function upgrade_5_66_alpha1($rev): void {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('Add fields to civicrm_mail_settings to allow more flexibility for email to activity', 'addMailSettingsFields');
+  }
+
+  /**
+   * Add fields to civicrm_mail_settings to allow more flexibility for email to activity
+   *
+   * @param \CRM_Queue_TaskContext $ctx
+   *
+   * @return bool
+   */
+  public static function addMailSettingsFields(CRM_Queue_TaskContext $ctx) {
+    self::addColumn($ctx, 'civicrm_mail_settings', 'is_active', 'tinyint NOT NULL DEFAULT 1 COMMENT "Ignored for bounce processing, only for email-to-activity"');
```

Comment:
Minor suggestion: If the upgrade is adding multiple columns in one step, suggest adding a `$ctx->log->info('Adding field ..');` in between each one to make it easier to troubleshoot where upgrades fail.

Reply:
Added logging for each step